### DISCUSSION TITLE
Re-Build Project then get error, please help me!

### DISCUSSION THREAD

**Discussion by JeNetwork** [mkaring/ConfuserEx#416] [Q&A]
Hello,

Re-Build Project then get error, please help me!
![2021-10-02_012413](https://user-images.githubusercontent.com/33093866/135673171-9f1236bf-c72d-4e13-ac7b-b1fda9b1dbec.png)

and Output:
![2021-10-02_012450](https://user-images.githubusercontent.com/33093866/135673201-dfad33ce-664d-4acd-9093-89b90f4a869f.png)

**Comment by mkaring** [comment]
I can't reproduce this problem. The build works fine for me.

**Comment by KvanTTT** [comment]
It looks like there is some issue related to dotnet framework (error due to dotnet 5). What versions do you have installed?

**Reply by JeNetwork** [reply]
My installed:
![2021-10-03_030506](https://user-images.githubusercontent.com/33093866/135730468-f305c6bf-9523-417b-b268-691c499a3b33.png)
![2021-10-03_030532](https://user-images.githubusercontent.com/33093866/135730477-efa32585-12d1-4bd5-b0ba-2fc275f7a04a.png)
![2021-10-03_030539](https://user-images.githubusercontent.com/33093866/135730482-f6b7f5fc-abab-4e67-8317-4996170dece2.png)

**Reply by mkaring** [reply]
I'm building with the exact same version and it's all good. NET 5.0.401.
Is this happening consistently for you? Or only during rebuild or incremental build?
Have you tried clearing out the cache of Visual Studio for the repository (deleting the `.vs` directory)?

**Reply by JeNetwork** [reply]
Let me try again! Thanks @mkaring

**Reply by JeNetwork** [reply]
Ah! @mkaring and @KvanTTT 

Please tell me this one: I can modify some thing in **Constant.cs** and **Mutation.cs** like pictures below, or please tell me some place which I can modify code as my custom private options. Thanks so much:
![2021-10-03_190105](https://user-images.githubusercontent.com/33093866/135752882-51c5742f-783a-435f-a2e5-a3db13bb668b.png)
![2021-10-03_190129](https://user-images.githubusercontent.com/33093866/135752884-6d979567-df7e-4d6b-b470-74d2be9359ca.png)

**Reply by mkaring** [reply]
You can change the values for the Mutation keys, but that is not useful, as those values are never used. You can set them to any value you like, or to the same value. Doesn't matter. References to these fields are overwritten during injection with values randomly generated during the obfuscation.

Changing the transformation function for the constant value is possible. You also need to change the counter part in the encode phase: [here](https://github.com/mkaring/ConfuserEx/blob/master/Confuser.Protections/Constants/EncodePhase.cs#L96-L98)

**Reply by JeNetwork** [reply]
I got it, thanks so much!
